docker/documentation-site-builder/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ruby:3.1-slim-bullseye as jekyll
+
+SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    build-essential \
+    git \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /docs
+
+# First copy the Gemfile
+COPY docs/Gemfile /docs/
+COPY docs/Gemfile.lock /docs/
+
+RUN gem update --system && gem install jekyll && gem install just-the-docs && gem cleanup
+
+# Copying the docs last, to maximize docker cache 
+COPY docs/ /docs/
+
+EXPOSE 4000
+
+# CMD [ "bundle", "exec", "jekyll", "serve", "--force_polling", "-H", "0.0.0.0", "-P", "4000" ]
+CMD ["bundle", "exec", "jekyll", "serve", "-H" , "0.0.0.0", "-P", "4000" ]
```

Comment:
Split this in two:

`ENTRYPOINT`: `bundle exec jekyll`
`CMD`: the rest.

Reply:
Done